README.md

```diff
@@ -33,6 +33,8 @@ spec:
 Start a local development environment with Kind where `crossplane` is installed:
 
 ```
+git submodule init
+git submodule update
```

Comment:
We actually have a make target (`fallthrough`) doing this which should be the first target in the makefile which is not the case currently. It is invoked if no target specified (e.g. just `make`)

So, we should better instruct to call `make` here and move `local-dev` target down in the Makefile as below:

```
diff --git a/Makefile b/Makefile
index 6cdd9a8..773d903 100644
--- a/Makefile
+++ b/Makefile
@@ -49,7 +49,6 @@ IMAGES = provider-kubernetes provider-kubernetes-controller
 # Setup Local Dev
 -include build/makelib/local.mk

-local-dev: local.up local.deploy.crossplane
 # ====================================================================================
 # Targets

@@ -82,6 +81,8 @@ generate.done: crds.clean
 # integration tests
 e2e.run: test-integration

+local-dev: local.up local.deploy.crossplane
+
 # Run integration tests.
 test-integration: $(KIND) $(KUBECTL) $(HELM3)
        @$(INFO) running integration tests using kind $(KIND_VERSION)
```

Reply:
Done, thanks @turkenh